lib/src/analyzer_impl.dart

```diff
@@ -122,8 +125,13 @@ class AnalyzerImpl {
   void prepareErrors() {
     for (Source source in sources) {
       context.computeErrors(source);
+
       var sourceErrors = context.getErrors(source);
       errorInfos.add(sourceErrors);
+
+      if (options.strongMode) {
+        errorInfos.add(strongChecker.computeErrors(source));
```

Comment:
alright. I'm okay either way, but here's an example of where having our own pass felt a bit silly: https://github.com/dart-lang/dev_compiler/commit/9738d60ad922ac9a2fa7c4fca64d1a13a844d279#diff-c341469b75ef94d504ccbc25d4f08f2cR445 ... I was adding boolean conversion checks, because our type rule is stronger, even though I'm 99% sure this same logic exists already in Analyzer (note: ignore the AST mutation in that CL, it's gone now: https://github.com/dart-lang/dev_compiler/commit/3acd63a6a4ebdde64c81da6a3d410b4b4b6fedd4)

Conceptually, all CodeChecker is doing is assignment checks, but with stronger subtype relation. Same thing for OverrideChecker. I suspect we could delete most of that file, if we had a couple of methods to override to provide the typing rules.

(It would probably fix some bugs in DDC too, for case we haven't considered, but y'all have, since Analyzer production readiness >>>>> DDC at this point :) )